app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -28,8 +30,7 @@ def access_token(input)
 
         # Validate ID Token is active
 
-        # Retrieve mail from response
-        conjur_username = "alice"
+        conjur_username = conjur_username(request_body)
```

Comment:
You could pass `request_body.id_token` into this method since the method doesn't need anything else from the request body.

Reply:
done